### PR TITLE
1905-09-11 issue verification (Joshua Kendrick)

### PR DESCRIPTION
**Pull request by Josh-LK**

### CODE REVIEW

**Comment by huoyanzhizhou** [comment]
Good start. Maybe you can check more about measures and features.

File: 1905-09-11.xml [hunk 1]
```diff
@@ -33,8 +34,8 @@
                 <div type="nameplate">
                     <head>THE EGYPTIAN GAZETTE</head>
                     <!-- September 11, 1905. -->
-                    <head type="sub">No. <measure quantity="7239">7,239</measure>] ALEXANDRIA,
-                        THURSDAY, JULY 13, 1905. [SIX PAGES P.T. 1. </head>
+                    <head type="sub">No. <measure quantity="7239">7,290</measure>] ALEXANDRIA,
+                        MONDAY, JULY 11, 1905. [SIX PAGES P.T. 1. </head>
```
Comment (whanley) [inline 1905-09-11.xml]: SEPTEMBER

File: 1905-09-11.xml [hunk 2]
```diff
@@ -1512,7 +1514,7 @@
                     <p>An English Daily Newspaper, Established in 1880.</p>
                     <p>Editor &amp; Manager: R. Snelling.</p>
                     <p>Price: One Piastre Tariff.</p>
-                    <p><date when="1905-07-13">THURSDAY, JULY 13, 1905</date>.</p>
+                    <p><date when="1905-07-13">MONDAY, JULY 11, 1905</date>.</p>
```
Comment (whanley) [inline 1905-09-11.xml]: SEPTEMBER

**Comment by whanley** [comment]
For deliverable 1, you're supposed to do Oct 10 1906, not 1905. Did you just misname the file?

**Comment by whanley** [comment]
How are things going, Josh? Any work to submit?

**Comment by whanley** [comment]
Took the 1906-10-03 issue manually, fixed the name, uploaded directly.

**Pull request closed without merging**